src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -608,6 +608,44 @@ public function firstOr($columns = ['*'], Closure $callback = null)
         return $callback();
     }
 
+    /**
+     * Create record in database and call user given callback with created model.
+     *
+     * @param  array  $attributes
+     * @param  Closure  $callback
+     * @return mixed|null
```

Comment:
`mixed` cannot exist next to any other type in a union. `null` is already contained in `mixed`.

Reply:
> Why not just use `tap()`?
> 
> ```
> \tap(new Model, function ($model) {
>  // do something ...
> });
> ```
> 
> reference: https://laravel.com/docs/9.x/helpers#method-tap
I think this is more user friendly and can help new users and actually it is obvious what it does.

Reply:
> `mixed` cannot exist next to any other type in a union. `null` is already contained in `mixed`.

yes I should fix it.